flutter/documentation/env-setup-windows.md

```diff
@@ -19,11 +19,13 @@ However, using Chocolatey greatly simplifies installation.
 * Install dependencies:
   * `choco install git -y`
   * `choco install make -y`
-  * `choco install bazel -y`
+  * `choco install bazel -y --version 4.2.1`
```

Comment:
Can you add a new section `Tested enviroment` instead of specify the version for only one tool? Else, the version for all other tools should be specified to be consistent.
You can see the example for iOS here:
https://github.com/mlcommons/mobile_app_open/blob/master/flutter/documentation/env-setup-ios.md#tested-environment